### PR TITLE
WRO-8765: Storybook: migrate default samplers from knobs to controls - LabeledIcon--> VirtualList

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed the stories from default(LabeledIcon to VirtualList) to work with controlls.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-8765

### CODE REVIEW

File: samples/sampler/stories/default/RangePicker.js [hunk 1]
```diff
@@ -1,52 +1,68 @@
...
+	}
+};
+
+export const _RangePicker = (args) => (
+	<RangePicker
+		decrementIcon={args['decrementIcon']}
+		defaultValue={0}
+		disabled={args['disabled']}
+		incrementIcon={args['incrementIcon']}
+		onChange={action('onChange')}
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/default/RangePicker.js]: Can you sort the props here please?

File: samples/sampler/stories/default/Picker.js [hunk 2]
```diff
@@ -1,51 +1,63 @@
...
+		wrap={args['wrap']}
+	>
+		{airports}
+	</Picker>
+);
+
+boolean('disabled', _Picker, Picker);
+boolean('joined', _Picker, Picker);
+boolean('noAnimation', _Picker, Picker);
+boolean('wrap', Picker);
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/default/Picker.js]: This controls is missing the story name and the control is not showing up
```suggestion
boolean('wrap', _Picker, Picker);
```

File: samples/sampler/stories/default/Picker.js [hunk 3]
```diff
@@ -1,51 +1,63 @@
...
+select('decrementIcon', _Picker, ['', ...decrementIcons], Picker);
+select('incrementIcon', _Picker, ['', ...incrementIcons], Picker);
+select('orientation', _Picker, prop.orientation, Picker, prop.orientation[0]);
+select('width', prop.width, Picker, prop.width[3]);
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/default/Picker.js]: Same here
```suggestion
select('width', _Picker, prop.width, Picker, prop.width[3]);
```

File: samples/sampler/stories/default/Popup.js [hunk 4]
```diff
@@ -1,37 +1,50 @@
...
+			onHide={action('onHide')}
+			onShow={action('onShow')}
+			open={args['open']}
+			scrimType={args['scrimType']}
+			showCloseButton={args['showCloseButton']}
+			spotlightRestrict={args['spotlightRestrict']}
+		>
+			<div>{args['children']}</div>
+		</Popup>
+		<BodyText centered>Use KNOBS to interact with Popup.</BodyText>
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/default/Popup.js]: Can you change the text inside the BodyText from KNOBS to Controls please?

File: samples/sampler/stories/default/ProgressBar.js [hunk 5]
```diff
@@ -1,49 +1,62 @@
...
+			) : null}
+		</ProgressBar>
+	);
+};
+
+boolean('disabled', _ProgressBar, ProgressBarConfig);
+boolean('highlighted', _ProgressBar, ProgressBarConfig);
+boolean('tooltip', _ProgressBar, ProgressBarTooltipConfig);
+number('backgroundProgress', _ProgressBar, ProgressBarConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.5);
+number('progress', _ProgressBar, ProgressBarConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.4);
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/default/ProgressBar.js]: Change the progress and backgroundProgress control from number to range. It seems that with number is not working properly

File: samples/sampler/stories/default/Slider.js [hunk 6]
```diff
@@ -1,58 +1,76 @@
...
+		</Slider>
+	);
+};
+
+boolean('activateOnFocus', _Slider, SliderConfig);
+boolean('disabled', _Slider, SliderConfig);
+boolean('noFill', _Slider, SliderConfig);
+boolean('percent', _Slider, SliderTooltipConfig);
+boolean('tooltip', _Slider, SliderTooltipConfig);
+number('backgroundProgress', _Slider, SliderConfig, {range: true, min: 0, max: 1, step: 0.01}, 0.5);
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/default/Slider.js]: Change control from number to range please

**Review by daniel-stoian-lgp — COMMENTED** [review]
approved

**Review by vladut-clapou-lgp — COMMENTED** [review]
LGTM

**Pull request merged**